appserver/ejb/ejb-container/src/main/java/com/sun/ejb/codegen/Generator.java

```diff
@@ -189,7 +206,7 @@ protected Method[] removeRedundantMethods(final Method[] methods) {
                 nodups.add(method);
             }
         }
-        return nodups.toArray(new Method[nodups.size()]);
```

Comment:
Let's do that the best now:
`nodups.toArray(Method[]::new)`

Reply:
You're right! Thank you for advice. Fixed.